Http/Controllers/Backend/Advanced/BatchesController.php

```diff
@@ -15,11 +15,10 @@ public function __construct()
     //----------------------------------------------------------
     public function getAssets(Request $request): JsonResponse
     {
-        if (!Auth::user()->hasPermission('has-access-of-advanced-section')) {
-            $response['success'] = false;
-            $response['errors'][] = trans("vaahcms::messages.permission_denied");
+        $permission_slug = 'has-access-of-advanced-section';
```

Comment:
@we-vikram-wri231 this should be array instead of line number 21, make this change everywhere else as well